src/include/CMakeLists.txt

```diff
@@ -49,6 +49,17 @@ install (FILES ${detail_headers}
          DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}/detail
          COMPONENT developer)
 
+file (GLOB flip_headers OpenImageIO/FLIP/*.h)
+install (FILES ${flip_headers}
+         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}/flip
+         COMPONENT developer)
+
```

Comment:
The flip headers don't need to be part of OIIO's public APIs, right?

So don't put them in src/include/OpenImageIO, and don't include them in the "install" list.

Just put the flip code in src/libOpenImageIO/flip and call it a day. The only code that needs to be aware of it is in some of the IBA modules in libOpenImageIO.